cql3/Cql.g

```diff
@@ -1336,6 +1338,21 @@ listServiceLevelAttachStatement returns [std::unique_ptr<list_service_level_atta
       { $stmt = std::make_unique<list_service_level_attachments_statement>(); }
     ;
 
+/**
+ * opis
+ */
+describeStatement returns [std::unique_ptr<cql3::statements::raw::describe_statement> stmt]
+    @init {
+        bool fullSchema = false;
+        bool pending = false;
+        bool config = false;
+        bool only = false;
+    }
+    : ( K_DESCRIBE | K_DESC )
+    ( (K_KEYSPACES) => K_KEYSPACES       { $stmt = cql3::statements::raw::describe_statement::keyspaces(); }
+    | (K_TABLES) => K_TABLES             { $stmt = cql3::statements::raw::describe_statement::tables();    }
+    );
```

Comment:
Many other types of DESCRIBE are missing in this implementation as I noted above. I'm not saying you have to do all of them in the first version, but please at least add a FIXME comment on what's missing.

Reply:
Also missing is `(K_WITH K_INTERNALS)?` at the end.

Reply:
I've implemented all the other types, except for generic `DESCRIBE <name>` but I'll add it soon.